### ISSUE TITLE
Request: More autofixes for `duplicate-literal-member`/`PYI062`

### ISSUE THREAD

**Issue by Avasam** [astral-sh/ruff#14187]
* A minimal code snippet that reproduces

https://docs.astral.sh/ruff/rules/duplicate-literal-member/#duplicate-literal-member-pyi062 mentions that
> Fix is sometimes available.

However cases like this which *looks* trivial is detected but not autofixed.
```py
def foo(bar: Literal["a", "b", "b"]): ...
```

* The current Ruff version (`ruff --version`).
ruff 0.7.2

**Comment by charliermarsh** [comment]
I actually don't think fix is ever available here; it's incorrectly marked as fixable, but we never create fixes for it.
